docs/src/user/guide/index.md

```diff
@@ -6,6 +6,8 @@ To switch to dark mode, simply go to your map and click on the little sun icon w
 
 ![Dark Mode 1](img/dark-mode-1.png)
 
+![test](img/print-3.png)
+
```

Comment:
Can this test be removed from the guide, please?

Reply:
Thanks Jeremy completely forgot about that when I was showing Sandhya will remove in PR